### DISCUSSION TITLE
Error "Hash collision!!! Fatal error!!" starting SDRPlusPlus

### DISCUSSION THREAD

**Discussion by MaxHusky94** [AlexandreRouma/SDRPlusPlus#148] [Q&A]
On starting SDR++ (built on Ubuntu 21.04. using the latest code) I get 

"Module 'sdrplay_source' doesn't exist" and 

"[INFO] [UHD] linux; GNU C++ version 10.2.1 20201207; Boost_107400; UHD_3.15.0.0-4build1

Hash collision!!! Fatal error!!"". 

With Ubuntu 20.10 (and a former version of the code) it worked flawlessly. Any suggestions?

Peter

**Comment by AlexandreRouma** [comment]
Hello
You likely installed the `soapysdr-module-all` (or similar) which is a know bad package on ubuntu. Uninstall everything linked to SoapySDR and reinstall only the soapy modules you need.

**Reply by AlexandreRouma** [reply]
by the way, this sort of questions are better suited to the "Issues" section as other may find it helpful is they get the same error.
